app/notifications/notifications_letter_callback.py

```diff
@@ -71,5 +71,7 @@ def process_letter_callback():
         notification_id = signing.decode(token)
     except BadSignature:
         current_app.logger.info("Letter callback with invalid token of %s received", token)
+    else:
+        current_app.logger.info("Letter callback for notification id %s received", notification_id)
 
-    current_app.logger.info("Letter callback for notification id %s received", notification_id)
+    return "ok"
```

Comment:
might be nice to say `return jsonify(result="success"), 200` in line with our SMS callbacks

Reply:
Sure, I was assuming that everything in this function would be overwritten when the code for it is written "properly" but have changed it